cloudigrade/util/tests/test_aws.py

```diff
@@ -29,7 +29,7 @@ def test_extract_account_id_from_arn(self):
     def test_error_extract_account_id_from_invalid_arn(self):
         """Assert error in account ID extraction from a badly-formed ARN."""
         mock_arn = faker.Faker().text()
-        with self.assertRaises(Exception):  # TODO more specific exceptions
```

Comment:
Deleting a `TODO` without actually fixing it? That's a paddlin'. 🏏

Reply:
I thought about it, but what does that `TODO` really accomplish? What exceptions do we want to check? With how boto3 handles them, is it even worth pursuing further?

Reply:
On second review, that test didn't actually have anything to do with boto3, so I went ahead and expanded it a tad.